docs/source/guides/search/endpoints.rst

```diff
@@ -132,6 +147,7 @@ The query syntax follows the rules:
 * **{comparison operator}** are eq, ne, gt, lt, ge, le
 * **{literal value}** is either a string between ``"`` (double quotes) or a numerical value (float or integer).
 * Wildcard searching is available with the **like** operator. The wildcarding syntax of the **{literal value}** follows the [OpenSearch Simple Query String](https://opensearch.org/docs/latest/opensearch/query-dsl/full-text/#simple-query-string) convention.
+* **{group}** has mandatory brackets which make any complex query loaded with brackets, as seen in the example above, don't forget them !
```

Comment:
request: replace "brackets" `[]` with "parentheses" `()` for correctness (and check for other erroneous usage of "brackets" in files in-scope for this pr

Reply:
I made that change, I noticed also that parentheses are sometimes called round brackets. Thanks @alexdunnjpl